source/index.html

```diff
@@ -35,7 +35,7 @@ <h1>The <b>Perl&nbsp;6</b> Programming Language</h1>
       <p>Hi, my name is Camelia.  I'm the spokesbug for Perl&nbsp;6,
         the plucky little sister of <a href="https://www.perl.org/">Perl&nbsp;5</a>.
         Like her world-famous big
-        sister, Perl&nbsp;6, now nicknamed "Raku", intends to carry forward the high ideals of
+        sister, Perl&nbsp;6, now nicknamed by some to be "Raku", intends to carry forward the high ideals of
```

Comment:
What about something neutral, like:

> Perl 6, also known as “Raku”, …

Or maybe it is not more neutral?

Reply:
My intention in saying "nicknamed by some" was to address the fact that not all people like the nickname. It was an attempt to be neutral or at least more honest. 

It's clearly not wise to merge anything right yet and to take time over over the exact wording.